nautobot/docs/installation/nautobot.md

```diff
@@ -85,13 +85,26 @@ $ echo $NAUTOBOT_ROOT
 
 ## Understanding the Virtual Environment
 
+!!! important
+    You may need to start a new terminal session to see the expected environment results below.
+
 Because the `nautobot` user was created with `NAUTOBOT_ROOT` set as its home directory and we had you set the shell to `/bin/bash`, the binary path `$NAUTOBOT_ROOT/bin` is automatically added to the beginning of the `$PATH` environment variable:
 
+In Ubuntu 20.04:
 ```no-highlight
 $ echo $PATH
 /opt/nautobot/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/snap/bin
 ```
 
+Due to differences between OS, in CentOS 8.2.2004 `$PATH` will appear as:
+```no-highlight
+$ echo $PATH
+/opt/nautobot/.local/bin:/opt/nautobot/bin:/opt/nautobot/.local/bin:/opt/nautobot/bin:/sbin:/bin:/usr/sbin:/usr/bin:/usr/local/sbin
```

Comment:
Double `PATH` prepends here? That is weird. Drop the version number for CentOS. please.

Reply:
Yes, Daniel Himes got that same CentOS path result as well.

Reply:
also dropped centos version